apps/chat/src/components/Chatbar/Conversation.tsx

```diff
@@ -686,6 +701,26 @@ export const ConversationComponent = ({ item: conversation, level }: Props) => {
           }}
         />
       )}
+      <ConfirmDialog
+        isOpen={isConfirmRenaming}
+        heading={t('Confirm renaming')}
+        confirmLabel={t('Rename')}
+        cancelLabel={t('Cancel')}
+        descriptionClasses="text-secondary"
+        textClasses="text-start"
+        buttonsClasses="w-full justify-end"
```

Comment:
please do this for all confirmation modals (inside modal component) to prevent incosistency